src/main.cpp

```diff
@@ -298,6 +298,11 @@ void setup()
     digitalWrite(VEXT_ENABLE, VEXT_ON_VALUE); // turn on the display power
 #endif
 
+#if defined(BIAS_T_ENABLE)
```

Comment:
Is a BIAS_T likely to be included in future variants? Otherwise, is this better in variant-specific code to keep main clean?

Reply:
There are 2 kinds of BIAS-T on the board. Once supplies a switchable DC offset (14 or 18V) to power an active antenna and switch its polarity. we generally want that off. The other possibility is a 5V DC offset on the antenna port to power an active antenna (also possible for GPS). So i'd leave that in without additional guards as it could be useful for other board designs. I used existing definitions for those switches where possible and added just those that were'nt available with existing methods.

These beasts have 6 GPIO's controlling the switching (!) :-)